cypress/e2e/safe-apps/permissions_settings.cy.js

```diff
@@ -1,36 +1,41 @@
 import * as constants from '../../support/constants'
+import * as main from '../pages/main.page'
+import * as safeapps from '../pages/safeapps.pages'
 
 let $dapps = []
+const app1 = 'https://app1.com'
+const app3 = 'https://app3.com'
 
 describe('The Safe Apps permissions settings section', () => {
   before(() => {
+    cy.clearLocalStorage()
     cy.on('window:before:load', (window) => {
       window.localStorage.setItem(
         constants.BROWSER_PERMISSIONS_KEY,
         JSON.stringify({
-          'https://app1.com': [
+          app1: [
```

Comment:
Let's keep the URL keys here to be consistent. You can use the variables as object keys to keep things concise:
```suggestion
          [app1]: [
```

Reply:
I won't mark the other changes here.